### PR TITLE
Reduce the number of times hack/update-codegen.sh is executed

### PR DESCRIPTION
**Pull request by bobdonat**
This pull request reduces the number of times hack/update-codegen.sh is executed when using the make target manfests.  The end result is that the make target runs faster and the clientsets have a different package name.  Updated the tests to use the new package name.

### CODE REVIEW

File: tests/e2e/pkg/metrics.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"net/http"
 
 	promoperapi "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
-	vaoClient "github.com/verrazzano/verrazzano/application-operator/clients/app/clientset/versioned"
+	vaoClient "github.com/verrazzano/verrazzano/application-operator/clientset/versioned"
```
Comment (pfmackin) [inline tests/e2e/pkg/metrics.go:13]: How do we distinguish between the client sets of the multiple apis?  Are they all merged into a single client set?   Same question for VPO

**Comment by pfmackin** [comment]
Does this require release note?

**Comment by bobdonat** [comment]
> Does this require release note?

Possibly, we don't document these but I guess someone could find them and use them.

File: tests/e2e/pkg/metrics.go (hunk 1 above)
Comment (bobdonat) [reply tests/e2e/pkg/metrics.go:13]: Yes, all VAO apis are merged into one clientset.  Same with VPO.

**Review by pfmackin — APPROVED** [review]

**Pull request merged**